libsolidity/ast/Types.cpp

```diff
@@ -665,15 +668,15 @@ string FixedPointType::toString(bool) const
 bigint FixedPointType::maxIntegerValue() const
 {
 	bigint maxValue = (bigint(1) << (m_totalBits - (isSigned() ? 1 : 0))) - 1;
-	return maxValue / pow(bigint(10), m_fractionalDigits);
+	return maxValue / boost::multiprecision::pow(bigint(10), m_fractionalDigits);
```

Comment:
This case didn't had a `using` statement, so not sure what `pow` did it pick up.

Reply:
This was argument-dependent lookup (https://en.cppreference.com/w/cpp/language/adl) - since ``bigint`` is in ``boost::multiprecision``, the ``pow`` of ``boost::multiprecision`` will be considered during lookup (and that's probably the only ``pow`` that can deal with ``bigint``s). But it's probably better to add the namespace explicitly - argument-dependent lookup is a somewhat weird concept...